packages/core/src/utilities/background-color/background-color.scss

```diff
@@ -260,3 +260,14 @@
 .ds-u-fill--transparent {
   background-color: transparent !important;
 }
+
+// Focus colors
+.ds-u-fill--focus-color {
```

Comment:
@line47 Do you think we need to add fill utilities for focus? I feel like its not necessary to provide utilities for every color, and I cant think of many use cases for this utility

Reply:
Currently, that's how the palette is generated. I'm open to suggestions for changing it

Reply:
hm I see. If thats the case we should also include it in the background color utilities page.